infra/terraform/base_stacks/dns_logging.tf

```
@@ -0,0 +1,10 @@
+resource "aws_cloudformation_stack" "dns-logging" {
```

A very minor thing: I generally follow the syntactic formatting of using underscores (`_`) in resource names. We then generally switch to hyphens for the name of the stack name as that is aws style. 



```suggestion
resource "aws_cloudformation_stack" "dns_logging" {
```